tests/fsharp/Compiler/Language/OptionalInteropTests.fs

```diff
@@ -6,15 +6,14 @@ open System.Collections.Immutable
 open NUnit.Framework
 open FSharp.Test
 open FSharp.Test.Utilities
-open FSharp.Test.Compiler
-open FSharp.Compiler.Diagnostics
 open Microsoft.CodeAnalysis
 
 [<TestFixture>]
 module OptionalInteropTests =
 
-    [<Test>]
-    let ``C# method with an optional parameter and called with an option type should compile`` () =
+    [<TestCase("5.0")>]
+    [<TestCase("latest")>]
```

Comment:
Should `preview` be added here?

Reply:
I thought about it, that'd be fine. The commit that introduces this change specifically mentions it's to test the compiler pre and post FS-1093 additional conversions but preview would be compatible with that wording.

Reply:
Changed to preview here too